Sources/Moya/Response.swift

```diff
@@ -124,11 +124,11 @@ public extension Response {
     func map<D: Decodable>(_ type: D.Type, atKeyPath keyPath: String? = nil, using decoder: JSONDecoder = JSONDecoder()) throws -> D {
         let jsonData: Data
         if let keyPath = keyPath {
-            guard let jsonDictionary = (try mapJSON() as? NSDictionary)?.value(forKeyPath: keyPath) as? [String: Any] else {
+            guard let jsonObject = (try mapJSON() as? NSDictionary)?.value(forKeyPath: keyPath) else {
                 throw MoyaError.jsonMapping(self)
             }
             do {
-                jsonData = try JSONSerialization.data(withJSONObject: jsonDictionary, options: [])
+                jsonData = try JSONSerialization.data(withJSONObject: jsonObject, options: [])
```

Comment:
Nit: you can omit the `options` parameter - it has a default value of `[]`.

Reply:
Oh didn't know that. It would be great!